File: rise_up2/lib/models/missions.dart
class Missions {
  int idMission;
  String missionName;
  String descriptionMission;
  DateTime dateMission;
  String locality;
  String stateMission;
  int financedAmount;


  Missions(
      this.idMission,
      this.missionName,
      this.descriptionMission,
      this.dateMission,
      this.locality,
      this.stateMission,
      this.financedAmount,);

  factory Missions.fromJson(Map<String, dynamic> json) {
    return Missions(
        json['id_mission'],
        json['mission_name'],
        json['description_mission'],
        DateTime.parse(json['date_mission']),
        json['locality'],
        json['state_mission'],
        json['financed_amount']
        );
        
  }
}

File: rise_up2/Rise_Up_API/Rise_Up_API/Rise_Up_API/Modules/Missions.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace Rise_Up_API.Modules
{
    public class Missions
    {
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        [Key] public int id_mission { get; set; }
        public string mission_name { get; set;}
        public string description_mission { get; set; }
        public DateTime date_mission { get; set; }
        public string locality { get; set; }
        public string state_mission { get; set; }
        public int financed_amount { get; set; }
        [JsonIgnore]
        public virtual ICollection<Missions_Collection_Points>? missions_collection_points { get; }
    }
}
